docs/running/cloud/cloud.rst

```diff
@@ -24,7 +24,7 @@ Toil can launch and manage a cluster of virtual machines to run using the *provi
 distributed over several nodes. The provisioner also has the ability to automatically scale up or down the size of
 the cluster to handle dynamic changes in computational demand (autoscaling).
 
-Currently we have working provisioners with AWS and GCE with Azure coming soon.
+Currently we have working provisioners with AWS and GCE. Azure provisioner works but has limited support and is unmaintained.
```

Comment:
Probably can't say it's `working`.

Something more like:
`The azure provisioner has limited support and is unmaintained.  We've left our (previously working) Azure support in place, but cannot guarantee it will work in the future as we no longer have the resources to continue testing Azure.`